src/screens/SettingsScreen.js

```diff
@@ -18,11 +25,111 @@ import {
 } from '../components';
 import { SettingsListItem } from '../components/SettingsListItem';
 import { arrowLeft } from '../icons';
+import { createMatomoUserId, removeMatomoUserId, storageHelper } from '../helpers';
+import { useMatomoTrackScreenView } from '../hooks';
+
+const { MATOMO_TRACKING } = consts;
 
 export const SettingsScreen = ({ navigation }) => {
   const { isConnected, isMainserverUp } = useContext(NetworkContext);
   const { orientation, dimensions } = useContext(OrientationContext);
   const [refreshing, setRefreshing] = useState(false);
+  const [sectionedData, setSectionedData] = useState([]);
+
+  useMatomoTrackScreenView(MATOMO_TRACKING.SCREEN_VIEW.SETTINGS);
+
+  useEffect(() => {
+    const updateSectionedData = async () => {
+      const { consent: matomoValue } = await storageHelper.matomoSettings();
+
+      setSectionedData([
+        {
+          data: [
+            {
+              title: texts.settingsTitles.pushNotifications,
+              topDivider: true,
+              type: 'toggle',
+              value: false
+            }
+          ]
+        },
+        {
+          data: [
+            {
+              title: texts.settingsTitles.analytics,
+              topDivider: true,
+              type: 'toggle',
+              value: matomoValue,
+              onActivate: (revert) =>
+                setTimeout(() => {
+                  Alert.alert(
+                    texts.settingsTitles.analytics,
+                    texts.settingsContents.analytics.onActivate,
+                    [
+                      {
+                        text: 'Nein',
+                        onPress: () => revert(),
```

Comment:
this should simply be
```ts
onPress: revert
```
I think

Reply:
true

Reply:
done in we need to pay attention on merging, because it needs to be done in a later case in `ToggleListItem` as well, as the file got split.